### DISCUSSION TITLE
Ignore JSON file from `gatsby-transformer-json`

### DISCUSSION THREAD

**Discussion by pettinarip** [gatsbyjs/gatsby#37554] [Help]
- Is there a way to ignore a specific JSON file from being parsed by `gatsby-transformer-json`?
- or instead, indicate what specific paths I want to be analyzed by the plugin?

**Comment by LekoArts** [comment]
Why do you need to be able to do that?

Anyways, I see two options:

- Don't source the file with `gatsby-source-filesystem`
- Change the mediaType so that the transformer doesn't run (it checks it here: https://github.com/gatsbyjs/gatsby/blob/86f33a54a53f886362d49762bb4f502527935bea/packages/gatsby-transformer-json/src/gatsby-node.js#L6)

**Reply by pettinarip** [reply]
Thanks @LekoArts.

The reason behind this is that, in our project, we are going to start using `gatsby-plugin-react-i18next` to handle translations. In this plugin, we are loading the locales (JSON files) using `gatsby-source-filesystem` and the plugin is already parsing those files itself. There is no need for `gatsby-transformer-json` to do that again.

Note that we have other JSON files in the project besides the locales that need to be parsed by `gatsby-transformer-json`.

That is why I was wondering if we could ignore specific paths from it to avoid wasting time and memory on parsing and building those nodes that are not going to be used.
